src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,81 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  searchMovie: (title: string) => void;
+  isMovieFound: boolean;
+  foundedMovie: Movie | null;
```

Comment:
Why do you need two of these? If movie is not found, the foundMovie var will have null value, thus meaning that movie hasn't been found
```suggestion
  foundedMovie: Movie | null;
```